ui/pages/api/store-signature.ts

```diff
@@ -6,7 +6,7 @@ import { provider } from '../../lib/connectors'
 import { domain, types, value } from '../../lib/sign-agreement'
 
 const client = new NFTStorage({
-  token: process.env.NEXT_PUBLIC_NFTSTORAGE_KEY || '',
+  token: process.env.NFTSTORAGE_KEY || '',
```

Comment:
@biendeveloper Would it be better to fetch this variable from `config.ts`?

Reply:
I think it is not required to move to config because the config file is accessible by the front end, better to leave it this way

Reply:
@biendeveloper Oh, okay, good point.  Thanks for clarifying 😄